server/models/Profile.js

```diff
@@ -0,0 +1,132 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+
+const profileSchema = new mongoose.Schema({
+  user_username: {
+    type: String,
+    required: true,
+  },
+  first_name: {
```

Comment:
Please use camelCase for field names - it's the standard naming convention in JS 

Ditto throughout!

Reply:
Initially I was planning to use camelCase, but I saw in the User model that the method was camelCase, but the schema fields were using underscores. For consistency, should I change the user model schema to camelCase as well or should I just do  the Profile model a little differently?

Reply:
Sure Chad, if you could update that as well. From this point on we'll preference `camelCase` 👍🏻